packages/sanity/src/core/releases/components/BundleMenuButton/BundleMenuButton.tsx

```diff
@@ -74,6 +82,12 @@ export const BundleMenuButton = ({disabled, bundle, documentCount}: BundleMenuBu
       ...bundle,
       archivedAt: isBundleArchived ? undefined : new Date().toISOString(),
     })
+
+    if (isBundleArchived) {
+      telemetry.log(UnarchivedRelease)
+    } else {
+      telemetry.log(ArchivedRelease)
+    }
```

Comment:
Comments from previous PR https://github.com/sanity-io/sanity/pull/7444#discussion_r1740679570

So technically it's:
- IsBundleArchived `true` -> it means that it's in the process of becoming false, so the event we want to track is unarchive
- IsBundleArchived `false` -> if means that it's in the process of becoming true, so the event we want to track is archive

Even the  `achivedAt: isBundleArchived ? undefined : new Date().toISOString(),` follows the same type of logic. If it's true then set it as undefined because technically it won't be archived anymore, if it's _not_ archived then add the date because it means it'll become archived

Reply:
Makes sense, thanks for clarifying

Reply:
I've added a comment here just to make sure that we have this in the future :)